extension/app/ts/components/simulationExplaining/SimulationSummary.tsx

```diff
@@ -646,3 +646,66 @@ export function SimulationSummary(param: SimulationSummaryParams) {
 		</div>
 	)
 }
+
+type ExtraDetailsTransactionCardParams = {
+	transaction: TransactionWithAddressBookEntries;
+}
+export function ExtraDetailsTransactionCard({ transaction }: ExtraDetailsTransactionCardParams) {
+	const [showSummary, setShowSummary] = useState<boolean>(false)
+	const textareaRef = useRef<HTMLTextAreaElement | null>(null)
+
+	useEffect(() => {
+		const resizeTextArea = () => {
+			if (textareaRef && textareaRef.current) {
+				textareaRef.current.style.height = '0px'
+				const scrollHeight = textareaRef.current.scrollHeight
+				textareaRef.current.style.height = scrollHeight + 'px'
+			}
+		}
+		resizeTextArea()
+		globalThis.addEventListener('resize', resizeTextArea)
+		return () => globalThis.removeEventListener('resize', resizeTextArea)
```

Comment:
This should really be done with CSS rather than JavaScript.  Not a blocker for this PR, but an issue should be created to fix it if this is merged as is.

Reply:
hmm, I feel it cannot be done with CSS? but I could be wrong. Maybe I have been looking at too old CSS examples 🤔